scripts/dashboard/dashboard.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-angular.module('app.dashboard', ['app.sonar']);
\ No newline at end of file
+angular.module('app.dashboard', ['app.sonar', 'app.gerrit']);
```

Comment:
I am not sure why this is not displaying this file correctly. Please view the file using the View button to see the change.